cmd/ocsp-responder/main.go

```diff
@@ -83,6 +87,43 @@ func newFilter(issuerCerts []string, serialPrefixes []string) (*ocspFilter, erro
 	return &ocspFilter{crypto.SHA1, issuerKeyHashes, issuerNameKeyHashes, serialPrefixes}, nil
 }
 
+type Responder struct {
```

Comment:
This type name is pretty confusing because there's also an `ocsp.Responder`.

Usually the way I would organize this sort of thing is: store clk, log, timeout directly on the dbSource as we did before; group together the metrics under a separate `metrics` object if there wind up being enough of them to become unwieldy.

Reply:
I removed responder in favor of a metrics specific type and constructor and moved clock, logging, and timeout back to the dbSource.